js/interface/add-markers.js

```diff
@@ -112,7 +112,7 @@ Fliplet.InteractiveMap.component('add-markers', {
         })
       }
 
-      this.dataSourceId = ds.id
+      this.markersDataSource.id = ds.id
```

Comment:
please confirm this doesn't break if "markersDataSource" is undefined

Reply:
@squallstar yes, your right i miss this point, i made changes and find another way